submissions/sadoharu/task_js_dom_/scripts.js

```diff
@@ -0,0 +1,53 @@
+window.addEventListener('DOMContentLoaded', () => {
+    const pages = document.querySelectorAll('.section__page');
+    const navList = document.querySelector('.nav-bar__list');
+
+    const navListItem = [
+        {title: 'Overview', icon: 'icon-browser'},
+        {title: 'About', icon: 'icon-about-dot-me'},
+        {title: 'Achievement', icon: 'icon-award'},
+        {title: 'Projects', icon: 'icon-files-empty'},
+        {title: 'Reviews', icon: 'icon-rate_review'}
+    ];
+
+    navListItem.forEach((item, i) => {
+        const liItem = document.createElement('li');
+        liItem.classList.add('nav-bar__list_item');
+        liItem.tabIndex = 0;
+        liItem.setAttribute('data-index', i);
+        liItem.innerHTML = `
+            <span class="nav-bar__list_icon ${item.icon}"></span>
+            <span class="nav-bar__list_title">${item.title}</span>
+            `;
+        navList.append(liItem);
+    });
+
+    const hideShowPage = (show = 0) => {
```

Comment:
```suggestion
    const togglePageVisibility = (show = 0) => {
```

Reply:
And what is `0` and why it's `show`? 🤔

Reply:
show - argument
0 - default parameter

https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Functions/Default_parameters

Reply:
I know what is arguments and I know what is `0` in this case. :)

The question is, why you are using them? Why do you need them? What is the idea behind them? Why it's a `0` and not a `boolean`?

Reply:
0 this is my page overview
1 is the about page

Reply:
3 - Achievement
4 - Projects
5 - Reviews

Reply:
So what you're doing is called [magic numbers](https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544). Please, create constant and then use it.

Reply:
no, its the html collection index